indexer/services/socks/__tests__/websocket/index.test.ts

```diff
@@ -60,9 +63,15 @@ describe('Index', () => {
     mockSub = new Subscriptions();
     invalidMsgHandlerSpy = jest.spyOn(InvalidMessageHandler.prototype, 'handleInvalidMessage');
     pingHandlerSpy = jest.spyOn(PingHandler.prototype, 'handlePing');
+    defaultRestricteCountries = config.RESTRICTED_COUNTRIES;
```

Comment:
why do you redefine this at the start of each test? Seems unnecessary instead of defining it as a const and initializing to `config.RESTRICTED_COUNTRIES` on line 35

Reply:
Moved.